src/wasm-s-parser.h

```diff
@@ -242,7 +242,7 @@ class SExpressionWasmBuilder {
       i++;
     }
   }
-  Name getLabel(Element& s);
+  Name getLabel(Element& s, bool isBranch = true);
```

Comment:
How about an enum to avoid the bool, like
```
enum class LabelType {
  Break
  Delegate
}
```
?

Reply:
Done